pages/mesosphere/dcos/1.13/administering-clusters/package-registry/index.md

```diff
@@ -326,4 +326,48 @@ See `dcos registry --help` for an exhaustive list of operations that you can use
 
 <p class="message--warning"><strong>WARNING: </strong>Removing a package while a service is still deployed may cause the service to stop working.</p>
 
-After executing the above instructions, the rest of the flow is identical to packages fetched from {{ model.packageRepo }}. The only difference is that you do not need Internet access (for customers with air-gapped environments) to install packages from `package-registry`.
+After executing the above instructions, the rest of the flow is identical to packages fetched from {{ model.packageRepo }}. The only difference is that you don't need Internet access (for customers with air-gapped environments) to install packages from `package-registry`.
+
+# Upgrading Package Registry from DC/OS 1.12 to later DC/OS versions
+Customers upgrading from DC/OS 1.12 to 1.13 or later will encounter issues with Package Registry. This issue only affects customers who have services deployed via Package Registry starting at DC/OS 1.12. Customers who deploy applications via Package Registry starting from DC/OS 1.13 or later aren't affected by this issue.
```

Comment:
This is awesome, thank you @kaiwalyajoshi!  However, I think there is still a missing use case here - if you installed the package registry with a non-default name in 1.12, you still MUST upgrade the service after upgrading to 1.13, because the package contents in the bootstrap registry changed and the old service will not be able to fetch the assets if/when it is restarted.

Reply:
I agree. AFAIK the issue of package registry not deploying after upgrade due to the bootstrap registry changing affects all versions of DC/OS 1.12 and later. Package registry must always be upgraded after upgrading DC/OS no matter what name it was installed with.

Reply:
Added another section around Package Registry requiring upgrades on each DC/OS version. Please review.